server.js

```diff
@@ -18,7 +21,8 @@ app.get('/', function (req, res) {
   res.render(__dirname + '/index', {'BASE_PATH_URL': process.env['BASE_PATH_URL']});
 });
 
-let port = process.env.PORT || 5000;
+// let port = process.env.PORT || 5000;
+let port = 5000;
```

Comment:
Any reason why you are not allowing the port to be set using `process.env.PORT`?

Reply:
There were conflicts with groups process.env.PORT, so the app didn't want to be run locally